styles.css

```diff
@@ -0,0 +1,40 @@
+#main-doc{
+    padding-left: 250px;
+    
+    }
+    #navbar{
+        padding:10px;
+        background-color: azure;
+        position: fixed;
+        top: 0px;
+        width: 200px;
+     
+        border: 2px solid black;
+        padding-right:40px;
+    }
+    
+    
+    a{
+        color: #1010ff;
+    }
+    
+    a:hover{
+        color: red;
+    }
+    
+    #introduction_page{
+        padding-left: 240px;
+        background-color: azure;
+    }
+    
+    .main-section{
+        background-color: azure;
+        padding: 50px 40px 100px 20px;
+        
+    }
+    
+    
+    .nav-link{
+        width: 100%;
+        background-color: #eee;
+    }
```

Comment:
Please kindly remove the underlined of the hyperlinks, using `text-decoration` property.